app/views/help/cookie_settings.html.haml

```diff
@@ -0,0 +1,59 @@
+- if params[:"analytical-cookies"] == "off"
+  .lcdui-notice-summary
```

Comment:
I would suggest using the green [Notification banner](https://design-system.service.gov.uk/components/notification-banner/) as recommended by the Design System [cookies page](https://design-system.service.gov.uk/patterns/cookies-page/)

Reply:
Done 👍